### PR TITLE
Add tests and fixes bugs

### PR DESCRIPTION
**Pull request by Zilula**
We added tests to reducers, actions, and selectors. Reference Trello card for further information.

### CODE REVIEW

File: src/actions/markdownActions.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { updateMarkdown, UPDATE_MARKDOWN } from './markdownActions';
+
+
+describe('Update markdown action', () => {
```
Comment (mehtaphysical) [inline src/actions/markdownActions.test.js]: I would name this something that will scale better as you add more actions. Maybe `markdown actions`?

Comment (Zilula) [reply src/actions/markdownActions.test.js]: Hello,

I have made changes to this PR based on your comments. 
Thank you,
Lance

File: src/actions/markdownActions.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { updateMarkdown, UPDATE_MARKDOWN } from './markdownActions';
+
+
+describe('test markdown actions', () => {
```
Comment (mehtaphysical) [inline src/actions/markdownActions.test.js]: make describe block messages consistent

File: src/reducers/markdownReducer.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import reducer from './markdownReducer';
+
+describe('markdown reducer tests', () => {
```
Comment (mehtaphysical) [inline src/reducers/markdownReducer.test.js]: make describe block messages consistent

File: src/selectors/markdownSelectors.test.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { getMarkdown } from './markdownSelectors';
+
+describe('selector tests', () => {
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.test.js]: make describe block messages consistent

File: src/actions/markdownActions.test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  it('updates a markdown file', () => {
```
Comment (mehtaphysical) [inline src/actions/markdownActions.test.js]: this doesn't update a markdown file. It creates an action that can be used to update a markdown file

File: src/selectors/markdownSelectors.test.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  it('test the get single markdown selector', () => {
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.test.js]: it test the get single markdown selector is not a good sentence. maybe

it gets markdown text from state.

File: src/selectors/markdownSelectors.test.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    const state = {
+      markdown: 'some string'
+    };
+    const markdown = getMarkdown(state);
+    expect(markdown).toEqual('some string');
+  });
+
+  it('it will return an empty state if state is empty', () => {
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.test.js]: it it will return an empty state if state is empty. maybe:

it gets empty markdown text

**Review by mehtaphysical — APPROVED** [review]

**Review by caripizza — APPROVED** [review]
🤠

File: src/reducers/markdownReducer.test.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
+import reducer from './markdownReducer';
+
+describe('markdown reducers tests', () => {
+
+  it('handles an update markdown action', () => {
+    const state = {
+      markdown: {
+        text: ''
```
Comment (MariahAdams) [inline src/reducers/markdownReducer.test.js:8]: Why is markdown nested state here?

Comment (caripizza) [reply src/reducers/markdownReducer.test.js:8]: We'll fix this next thanks Mariah 🥇

**Pull request merged**